extutils/maskfinder/__init__.py

```diff
@@ -1 +1,4 @@
+"""
```

Comment:
Codacy found an issue: [One-line docstring should fit on one line with quotes (found 3) (D200)](https://app.codacy.com/manual/RaenonX/Jelly-Bot/pullRequest?prid=6015490)